SimPPS/DirectSimProducer/python/profiles_2016_cff.py

```diff
@@ -0,0 +1,49 @@
+import FWCore.ParameterSet.Config as cms
+from SimPPS.DirectSimProducer.profile_base_cff import profile_base as _base
+from CalibPPS.ESProducers.ctppsOpticalFunctions_non_DB_cff import optics_2016_preTS2, optics_2016_postTS2
+
+# base profile settings for 2016
+_base_2016 = _base.clone(
+    ctppsLHCInfo = _base.ctppsLHCInfo.clone(
+        beamEnergy = 6500
+    )
+)
+
+profile_2016_preTS2 = _base_2016.clone(
+    L_int = cms.double(6.138092276 + 3.654039035),
+    ctppsLHCInfo = _base_2016.ctppsLHCInfo.clone(
+        xangleBetaStarHistogramObject = cms.string("2016_preTS2/h2_betaStar_vs_xangle")
+    ),
+    ctppsOpticalFunctions = _base_2016.ctppsOpticalFunctions.clone(
+        opticalFunctions = optics_2016_preTS2.opticalFunctions,
+        scoringPlanes = optics_2016_preTS2.scoringPlanes,
+    ),
+    ctppsRPAlignmentCorrectionsDataXML = _base_2016.ctppsRPAlignmentCorrectionsDataXML.clone(
+        MisalignedFiles = cms.vstring("Validation/CTPPS/alignment/2016_preTS2.xml"),
+        RealFiles = cms.vstring("Validation/CTPPS/alignment/2016_preTS2.xml")
```

Comment:
```suggestion
        MisalignedFiles = "Validation/CTPPS/alignment/2016_preTS2.xml",
        RealFiles = "Validation/CTPPS/alignment/2016_preTS2.xml"
```

Reply:
Even for a `cms.vstring`? Could you please point me to a reference page (twiki, README.md, ...) where this convention of dropping the `cms.` custom types is discussed?

Reply:
Indeed, vstring's have to be enclosed between squared parentheses `[""]`, as you already do for example in https://github.com/cms-sw/cmssw/pull/38280/files#diff-5f947ac2e57f292fd308037908a758504b5722d9c770bcf6f71258dff5986470R15 , and contrarily to what I wrote too quickly in a few lines of my comments.

About a possible reference for it, you can take item 4 in https://twiki.cern.ch/twiki/bin/view/CMS/OpenRecoTasks#Review_and_optimization_of_the_p